src/content/ausbildung/funketikette.svx

```diff
@@ -1,28 +1,39 @@
 ---
 title: Funketikette
-date: '2023-02-06'
+date: '2023-06-01'
 nav_index: 2
 ---
 
 <script lang="ts">
+    import Tooltip from "$lib/wikitools/Tooltip.svelte";
     import alphabet from '$lib/images/funketikette/alphabet.png';
-    import funk_aceself from '$lib/images/funketikette/funk_aceself.png';
-    import sem52sl from '$lib/images/funketikette/sem52sl.png';
     import Keys from "$lib/wikitools/Keys.svelte";
 </script>
 
 Grundvoraussetzung für eine gelungene Missionsdurchführung ist eine gute Kommunikation zwischen allen Kräften. Über längere Strecken und Einheiten hinweg nutzt man den Funk. Die Grundzüge von gutem Funken sollten allen Soldaten bekannt sein.
 
-Diese Anleitung vermittelt bloß die absoluten Grundlagen des Funkens. Je nach Spezialisierung sind noch weitere Dinge relevant, wie etwa beim Funken im Konvoi und zwischen der Logistik, im Funk unter Truppführern, Sanitätern oder Aufklärern. Dazu finden sich Einträge mit weiterführenden Informationen auf den jeweiligen Unterseiten.
+Diese Seite vermittelt die absoluten Grundlagen des Funkens. Zusätzliche Festlegungen beim Funken, wie zum Beispiel beim [Fahren im Konvoi](/ausbildung/fahren-im-konvoi#funken), befinden sich auf den jeweiligen Unterseiten.
 
 ## Vor dem Funken
 
 Noch bevor man beginnt, Funkkontakt aufzubauen, überlegt man sich genau, was man mitteilen möchte. Ein Sender blockiert für die Dauer seines Funkspruches den Funkkanal und damit möglicherweise andere eilige Meldungen. Darum sollte ein Funkspruch gut formuliert, knapp, aber unmissverständlich gehalten werden. Guter Funk folgt also der Devise: _So viel wie nötig, so kurz wie möglich._
 
+## Verbindung
+
+Das Herstellen des Funkkontakts nennt man auch Verbindung. Sie folgt einem einfachen Schema: Der Sender identifiziert sich und den Empfänger des Funkspruches. Danach wird in einem Wort erklärt, welchen Zweck die Mitteilung verfolgt. Zum Beispiel:
+
+„Aufklärung für Truppführung - Information - Infanterie in Gruppenstärke zu Fuß gesichtet nord-west ihrer Aktuellen - Kommen.“
```

Comment:
This could use an example box but I understand why we might choose not to use one here.